### ISSUE TITLE
next-auth error (CLIENT_FETCH_ERROR)

### ISSUE THREAD

**Issue by JaviTrek** [WarsWorld/WarsWorld#205]
Unsure why when loading certain pages, this logs
![image](https://github.com/WarsWorld/WarsWorld/assets/96269542/c77dc9e3-8f63-4e4b-9147-bb8ad8ae00af)

**Comment by cliveglitch** [comment]
Found this:
![image](https://github.com/WarsWorld/WarsWorld/assets/56766746/1c0b56c0-f733-45a9-9485-445bc85a601d)
https://stackoverflow.com/questions/76177919/why-do-i-get-next-autherrorclient-fetch-error-messageundefined

**Comment by JaviTrek** [comment]
@cliveglitch That would make so much sense, I tried implementing that but couldn't. Didn't know it could be the cause of it, will research more tomorrow unless you want to take on this!

**Comment by cliveglitch** [comment]
@JaviTrek  yeah, I also got problems implementing that, but I have the theory that because this is kind of a recent update trpc still doesn't support getServerSession so it could be unsolvable for now, needs a huge workaround or something else.

https://trpc.io/docs/server/context
Although considering trpc documentation they show an example with getSession, so maybe it's outdated or that isn't the problem.
